docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -125,6 +132,56 @@ show up in ``DESCRIBE <table-name>`` or ``SELECT *``.
 
 This property is optional; the default is ``true``.
 
+``kafka.security-protocol``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable ``SSL`` for the Kafka connection, set this property to ``SSL``.
+
+This property is optional; the default ist ``PLAINTEXT``.
+
+``kafka.keystore-location``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Location of the keystore file used for connection to Kafka cluster.
+
+This property is optional.
+
+``kafka.keystore-password``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Password for the keystore file used for connection to Kafka cluster.
+
+This property is optional.
```

Comment:
But required when `kafka.keystore-location`, is given. Same goes for `kafka.key-password` and truststores.